### ISSUE TITLE
About youtube usage notes

### ISSUE THREAD

**Issue by kernxg** [ytdl-org/youtube-dl#31146]
## Checklist

- [x ] I'm asking a question
- [x ] I've looked through the README and FAQ for similar questions
- [x ] I've searched the bugtracker for similar questions including closed ones

## Question

I would like to create and distribute a shell script (linux) that uses youtube-dl to extract links to the audio of videos and playlists in youtube, and then automatically send them to a player.

But I can't figure out if this is possible without breaking youtube's terms of use.
In other words, if I can do this without any problems.
